lightsaber/processor/src/main/java/com/joom/lightsaber/processor/generation/ProviderClassGenerator.kt

```diff
@@ -244,9 +245,14 @@ class ProviderClassGenerator(
     invokeConstructor(type, CONSTRUCTOR_WITH_INJECTOR)
   }
 
-  private fun GeneratorAdapter.provideFromContractProvisionPoint(contractType: Type.Object, contractProvisionPoint: ContractProvisionPoint) {
+  private fun GeneratorAdapter.provideFromContractProvisionPoint(
+    contractType: Type.Object,
+    converter: ImportPoint.Converter,
+    contractProvisionPoint: ContractProvisionPoint) {
```

Comment:
Please move the parenthesis to a new line.

Reply:
done